ubl/Handlers/Invoice/InvoiceLineHandler.php

```diff
@@ -23,9 +23,7 @@ public function handle( $data, $options = array() ) {
 			$line_tax_data    = $item[ $taxDataContainer ];
 
 			foreach ( $line_tax_data[ $taxDataKey ] as $tax_id => $tax ) {
-				if ( ! is_numeric( $tax ) ) {
-					continue;
-				}
+				$tax = absint( $tax );
```

Comment:
Could you clarify the reason for this change? It seems the tax value might be a float string, but you're reassigning it as an integer. Additionally, zero is still a numeric value, so I'm curious about the intent behind this adjustment.

Reply:
The `$tax` value was actually an empty string, and I wanted to get `0` if it's an empty string, but I forgot about float values. I'll update the code.

Reply:
For me, it wasn't an empty string; it was `'0'`. Have you checked it again? Wouldn't it be helpful to know what the validation error was?

Reply:
I understand. The problem isn’t that the item’s value is zero; it’s that no tax has been applied to it.